NEWS

```diff
@@ -16,6 +16,9 @@ the selector called "arclength" with the value in column
 allow configurable text size via
 theme(axis.title=element_text(size=20)) etc.
 
+2022.8.25
+
+suggest data.table, use for faster compilation.
```

Comment:
hey @Faye-yufan you should increment the version and add a small comment in NEWS before merging each PR.

Reply:
Got it!